OutOfSchool/OutOfSchool.DataAccess/Repository/RatingRepository.cs

```diff
@@ -38,4 +40,9 @@ public RatingRepository(OutOfSchoolDbContext dbContext)
             .GroupBy(rating => rating.EntityId)
             .ToDictionary(g => g.Key, g => Tuple.Create(g.Average(p => p.Rate), g.Count()));
     }
+
+    public Task<List<Rating>> GetPartAsync(int skip, int take)
```

Comment:
repository already has a "generic" `Get` method that accepts optional filters, skip & take.